src/Vm.sol

```diff
@@ -188,156 +249,135 @@ interface VmSafe {
     // Sets all subsequent calls' msg.sender to be the input address until `stopPrank` is called
-    function startPrank(address) external;
+    function startPrank(address msgSender) external;
     // Sets the *next* call's msg.sender to be the input address, and the tx.origin to be the second input
-    function prank(address, address) external;
+    function prank(address msgSender, address txOrigin) external;
     // Sets all subsequent calls' msg.sender to be the input address until `stopPrank` is called, and the tx.origin to be the second input
-    function startPrank(address, address) external;
+    function startPrank(address msgSender, address txOrigin) external;
     // Resets subsequent calls' msg.sender to be `address(this)`
     function stopPrank() external;
-    // Sets an address' balance, (who, newBalance)
-    function deal(address, uint256) external;
-    // Sets an address' code, (who, newCode)
-    function etch(address, bytes calldata) external;
+    // Sets an address' balance
+    function deal(address who, uint256 newBalance) external;
+    // Sets an address' code
+    function etch(address who, bytes calldata newCode) external;
     // Expects an error on next call
-    function expectRevert(bytes calldata) external;
-    function expectRevert(bytes4) external;
+    function expectRevert(bytes calldata revertData) external;
+    function expectRevert(bytes4 revertData) external;
     function expectRevert() external;
     // Prepare an expected log with (bool checkTopic1, bool checkTopic2, bool checkTopic3, bool checkData).
     // Call this function, then emit an event, then call a function. Internally after the call, we check if
     // logs were emitted in the expected order with the expected topics and data (as specified by the booleans)
-    function expectEmit(bool, bool, bool, bool) external;
-    function expectEmit(bool, bool, bool, bool, address) external;
+    function expectEmit(bool checkTopic1, bool checkTopic2, bool checkTopic3, bool checkData) external;
+    function expectEmit(bool checkTopic1, bool checkTopic2, bool checkTopic3, bool checkData, address emitter)
+        external;
     // Mocks a call to an address, returning specified data.
     // Calldata can either be strict or a partial match, e.g. if you only
     // pass a Solidity selector to the expected calldata, then the entire Solidity
     // function will be mocked.
-    function mockCall(address, bytes calldata, bytes calldata) external;
+    function mockCall(address callee, bytes calldata data, bytes calldata returnData) external;
     // Mocks a call to an address with a specific msg.value, returning specified data.
     // Calldata match takes precedence over msg.value in case of ambiguity.
-    function mockCall(address, uint256, bytes calldata, bytes calldata) external;
+    function mockCall(address callee, uint256 msgValue, bytes calldata data, bytes calldata returnData) external;
```

Comment:
```suggestion
    function mockCall(address target, uint256 msgValue, bytes calldata data, bytes calldata returnData) external;
```

Reply:
I like `callee` better in this context because it refers to the action of calling a contract. `target` is much more broad.